wire-ios-utilities/Source/Public/PrivateUserDefaults.swift

```diff
@@ -96,3 +100,22 @@ public protocol DefaultsKey {
     var rawValue: String { get }
 
 }
+
+extension Never: DefaultsKey {
+
+    public var rawValue: String { fatalError() }
+
+}
+
+extension PrivateUserDefaults where Key == Never {
```

Comment:
`Never` is required so that we can call this static method on this generic type without specifying the type of `Key` as we don't care.